packages/web-react/README.md

```diff
@@ -1,5 +1,7 @@
 # @lmc-eu/spirit-web-react
 
+[![minified](https://badgen.net/bundlephobia/min/@lmc-eu/spirit-web-react)](https://bundlephobia.com/package/@lmc-eu/spirit-web-react) [![minified + gzip](https://badgen.net/bundlephobia/minzip/@lmc-eu/spirit-web-react)](https://bundlephobia.com/package/@lmc-eu/spirit-web-react) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@lmc-eu/spirit-web-react)](https://bundlephobia.com/package/@lmc-eu/spirit-web-react) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@lmc-eu/spirit-web-react)](https://bundlephobia.com/package/@lmc-eu/spirit-web-react)
```

Comment:
Could we have each badge on its own line please? Just in the MD source, it's hard to read like this…

Reply:
My bad. I only tested this in the IDE preview where the new lines are not handled correctly. Fixed.